### PR TITLE
Release notes for 1.1.0 preview 2

### PR DESCRIPTION
**Pull request by David-Engel**

### CODE REVIEW

File: release-notes/1.1/1.1.0-preview2.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+- .NET Core 2.1+ (Windows x86, Windows x64, Linux, macOS)
+- .NET Standard 2.0+ (Windows x86, Windows x64, Linux, macOS)
+
+### Dependencies
+
+#### .NET Framework
+
+- System.Data.Common 4.3.0
+- Microsoft.Data.SqlClient.SNI 1.1.0
+- Microsoft.Identity.Client 3.0.8
```
Comment (karinazhou) [inline release-notes/1.1/1.1.0-preview2.md]: We now have more dependencies for enclave provider: 
- Microsoft.IdentityModel.Protocols.OpenIdConnect
- Microsoft.IdentityModel.JsonWebTokens

File: release-notes/1.1/1.1.0-preview2.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+- System.Buffers 4.4.0
+- System.Diagnostics.DiagnosticSource 4.5.0
+- System.Memory 4.5.1
+- System.Security.Principal.Windows 4.5.0
+- System.Text.Encoding.CodePages 4.5.0
+- System.Configuration.ConfigurationManager 4.5.0
+- Microsoft.Identity.Client 3.0.8
+
+### Always Encrypted with secure enclaves
+
```
Comment (karinazhou) [inline release-notes/1.1/1.1.0-preview2.md]: Probably we need to add how to use the enclave feature by specifying `Attestation Protocol` and `Enclave Attestation Url` in the connection string.
e.g. 
`Attestation Protocol = HGS; Enclave Attestation Url = <attestation_url_for_HGS>`
`Attestation Protocol = AAS; Enclave Attestation Url = <attestation_url_for_AAS>`

File: release-notes/1.1/1.1.0-preview2.md (hunk 1 above)
Comment (David-Engel) [reply release-notes/1.1/1.1.0-preview2.md]: Done

File: release-notes/1.1/1.1.0-preview2.md (hunk 2 above)
Comment (David-Engel) [reply release-notes/1.1/1.1.0-preview2.md]: Done

**Pull request merged**